device/arista/x86_64-arista_7060x6_64pe/Arista-7060X6-64PE-P32O64/buffers.json.j2

```diff
@@ -0,0 +1,2 @@
+{%- set default_topo = 't1' %}
```

Comment:
Please set `default_topo` to `lt2` so that default buffer configuration `buffers_default_lt2.j2` can be referenced